src/LessMsi.Gui/MainForm.cs

```diff
@@ -753,7 +767,7 @@ private void InitializeComponent()
             this.exitToolStripMenuItem.Name = "exitToolStripMenuItem";
             this.exitToolStripMenuItem.ShortcutKeys = ((System.Windows.Forms.Keys)((System.Windows.Forms.Keys.Alt | System.Windows.Forms.Keys.F4)));
             this.exitToolStripMenuItem.Size = new System.Drawing.Size(152, 22);
-            this.exitToolStripMenuItem.Text = "E&xit";
+            this.exitToolStripMenuItem.Text = $"{Strings.Exit}";

```

Comment:
Is the `&` in the translation in this one or did we lose the `Alt+X` keyboard accelerator ?

Reply:
The ```&``` is in both the english and italian translations:
![image](https://github.com/user-attachments/assets/ba9c662b-01a4-46b0-8b8b-942a25401eb4)


![image](https://github.com/user-attachments/assets/4fa5a551-baf5-4ae5-97b7-d15ec4f7ff8d)